build.sbt

```diff
@@ -40,14 +42,43 @@ val dep = {
   )
 }
 
+lazy val solidityCompileTask = TaskKey[Unit]("solidityCompile", "Compiles solidity contracts")
+
+solidityCompileTask := {
```

Comment:
could be, but it relies on `solc` installed in system, for me it kind of makes sense to have it on sbt level